examples/experimental/tensorflow/classification/main.py

```diff
@@ -0,0 +1,232 @@
+"""
```

Comment:
Why do we need an entire separate classification sample for this? Isn't the new algo supposed to be working with the original samples?

Reply:
Thanks for the comment! I need to build the `tf.Graph` outside the strategy scope. But I agree that the new algorithm should work with the original examples. I will investigate how to resolve this issue in the future.

Reply:
UPD: The problem which was mentioned above was fixed. But, I still need a separate sample because I should wrap the model to the `NNCFNetwork`.

Reply:
Could you wrap the model to the NNCFNetwork into create_compressed_model?